integrations.json

```diff
@@ -372,8 +372,8 @@
           "assembly": "MongoDB.Driver.Core",
           "type": "MongoDB.Driver.Core.WireProtocol.IWireProtocol",
           "method": "ExecuteAsync",
-          "minimum_major": 0,
-          "minimum_minor": 0,
+          "minimum_major": 2,
+          "minimum_minor": 1,
           "minimum_patch": 0,
           "maximum_major": 65535,
```

Comment:
Don't we need to set both the minimum and maximum versions to 2.1 so it only applies to that version?